### PR TITLE
fix: add option to decode unicode in order to prevent crash

### PR DESCRIPTION
**Pull request by tasola**
Solves https://github.com/privateOmega/html-to-docx/issues/130

As far as I've been able to test, it works fine. But please, do test it thoroughly!

### CODE REVIEW

**Comment by privateOmega** [comment]
@tasola Please resolve the conflicts in this. Thanks

**Comment by tasola** [comment]
@privateOmega Fixed! I also added a description of the flag in the README.

File: src/html-to-docx.js [hunk 1]
```diff
@@ -126,8 +127,13 @@ async function addFilesToContainer(
     // eslint-disable-next-line no-param-reassign
     footerHTMLString = defaultHTMLString;
   }
+  if (documentOptions.decodeUnicode) {
+    // eslint-disable-next-line no-param-reassign
+    htmlString = decode(htmlString);
```
Comment (privateOmega) [inline src/html-to-docx.js:132]: dont you want to do this for footer and header content as well if they are present?

**Comment by privateOmega** [comment]
@tasola Sorry for getting to this late, but could you please rebase instead of merge with develop, thats our default strategy.

**Comment by privateOmega** [comment]
Also please add html-entities into rollup-config's external array

**Comment by tasola** [comment]
@privateOmega Excuse my delay as well! Done!

**Pull request merged**